server/src/db.js

```diff
@@ -1,11 +1,8 @@
-let user = process.env.DB_USER || process.env.USER;
-let pass = process.env.DB_PASS;
-let host = process.env.DB_HOST || "localhost:5432";
-
-pass = pass ? ":" + pass : "";
+const config = require("./config").root();
```

Comment:
How does this work?  Why doesn't config.js export anything?  What is this .root() thing?